packages/mask-sdk/public-api/mask-wallet.ts

```diff
@@ -30,15 +34,21 @@ declare namespace Mask.Ethereum {
      * @see https://nodejs.org/api/events.html
      */
     export interface EventEmitter {
+        /**
+         * @remarks Since API=0
+         */
         on(eventName: string | symbol, listener: (...args: any[]) => void): this
+        /**
+         * @remarks Since API=0
+         */
         removeListener(eventName: string | symbol, listener: (...args: any[]) => void): this
     }
     export interface EthereumEventEmitter extends EventEmitter {
-        on(eventName: 'message', listener: (message: ProviderMessage | EthSubscription) => void): this
-        on(eventName: 'connect', listener: (message: ProviderConnectInfo) => void): this
-        on(eventName: 'disconnect', listener: (error: ProviderRpcError) => void): this
-        on(eventName: 'chainChanged', listener: (chainId: string) => void): this
-        on(eventName: 'accountsChanged', listener: (accounts: string[]) => void): this
+        // on(eventName: 'message', listener: (message: ProviderMessage | EthSubscription) => void): this
+        // on(eventName: 'connect', listener: (message: ProviderConnectInfo) => void): this
+        // on(eventName: 'disconnect', listener: (error: ProviderRpcError) => void): this
+        // on(eventName: 'chainChanged', listener: (chainId: string) => void): this
+        // on(eventName: 'accountsChanged', listener: (accounts: string[]) => void): this
```

Comment:
```suggestion
```

Reply:
those APIs are intended to be implemented in the future so I'd like to keep them.